hail/python/hail/genetics/pedigree.py

```diff
@@ -148,9 +132,33 @@ def is_complete(self):
         :rtype: bool
         """
 
-        if not hasattr(self, '_complete'):
-            self._complete = self._jrep.isComplete()
-        return self._complete
+        return self._pat_id is not None and self._mat_id is not None and self._is_female is not None
```

Comment:
I don't think `is_female` should be required here. The java implementation is this:

```
  def toCompleteTrio: Option[CompleteTrio] =		
     dad.flatMap(d =>		
       mom.map(m => CompleteTrio(kid, fam, d, m, sex)))
```

Reply:
Based on the Java code, I would agree. But in the docs we claim that sex should be defined for completeness. So I'm not sure which is correct. I based this on the docs without carefully reading the Java code it seems.

Reply:
We shouldn't have interface changes unless it's an obvious bug fix. I favor sticking with the original java code implementation and creating an issue for 0.3 to revisit this.